sdk/spring/spring-cloud-azure-autoconfigure/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.4.0-beta.1 (Unreleased)
+## 4.4.0 (2022-08-08)
 
 ### Features Added
```

Comment:
We should change the content later, once this PR gets merged.

Reply:
yes, I know